### DISCUSSION TITLE
記事の検索結果がちょうど10の倍数のときページネーションが1ページ分多く表示される

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#388] [要望・アイデア・不具合]
たとえば現在「title:LTSV」で検索すると50件の記事がヒットします（記事が増減して50件ではなくなっていたら、お手数ですが10の倍数になる他のクエリを探してください）が、ページネーションは6ページあることになっています。

https://qiita.com/search?q=title%3ALTSV

1ページあたり10件なので実際には5ページまでしかなく、6ページ目にアクセスすると次のようなエラーメッセージが表示されます。

>「title:LTSV」に一致する記事は見つかりませんでした。

https://qiita.com/search?q=title%3ALTSV&sort=&stocked=&page=6

ページ数の計算で1ページ分のずれがありそうです。

**Comment by tanaka-asami415** [comment]
@munierujp 
ご報告ありがとうございます🙏こちら調査して対応します！

**Reply by kyntk** [reply]
@munierujp  大変お待たせいたしました。
ページネーション数の不具合を解消しました！
https://qiita.com/search?q=rails%20created%3A%3E2023-04-09%20stocks%3A%3E1&sort=&stocked=&page=3
